src/cascadia/TerminalSettingsEditor/Actions.h

```diff
@@ -120,8 +131,10 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         void _ViewModelPropertyChangedHandler(const Windows::Foundation::IInspectable& senderVM, const Windows::UI::Xaml::Data::PropertyChangedEventArgs& args);
         void _ViewModelDeleteKeyBindingHandler(const Editor::KeyBindingViewModel& senderVM, const Control::KeyChord& args);
         void _ViewModelModifyKeyBindingHandler(const Editor::KeyBindingViewModel& senderVM, const Editor::ModifyKeyBindingEventArgs& args);
+        void _ViewModelDeleteNewlyAddedKeyBindingHandler(const Editor::KeyBindingViewModel& senderVM, const IInspectable& args);
 
         std::optional<uint32_t> _GetContainerIndexByKeyChord(const Control::KeyChord& keys);
+        void _RegisterEvents(com_ptr<implementation::KeyBindingViewModel> kbdVM);

```

Comment:
Does this need to be an owning pointer? You have a whole addref/release going on when you enter and leave this function thanks to the parameter being passed by value.